src/status_im2/contexts/chat/messages/content/reactions/view.cljs

```diff
@@ -17,25 +17,14 @@
                    :emoji-id   emoji-id}])))
 
 (defn- on-long-press
-  [message-id emoji-id]
+  [message-id emoji-id user-message-content reactions]
   (rf/dispatch [:chat.ui/emoji-reactions-by-message-id
-                {:message-id         message-id
-                 :long-pressed-emoji emoji-id}]))
-
-(defn show-authors-sheet
-  [user-message-content reactions]
-  (rf/dispatch [:dismiss-keyboard])
-  (rf/dispatch
-   [:show-bottom-sheet
-    {:on-close                (fn []
-                                (rf/dispatch
-                                 [:chat/clear-emoji-reaction-author-details]))
-     :content                 (fn []
-                                [drawers/reaction-authors
-                                 (map :emoji-id reactions)])
-     :selected-item           (fn []
-                                user-message-content)
-     :padding-bottom-override 0}]))
+                {:message-id           message-id
+                 :long-pressed-emoji   emoji-id
+                 :drawer-content       (fn []
+                                         [drawers/reaction-authors reactions])
+                 :user-message-content (fn []
+                                         user-message-content)}]))
```

Comment:
```clojure
(rf/dispatch [:chat.ui/emoji-reactions-by-message-id
                {:message-id           message-id
                 :long-pressed-emoji   emoji-id
                 :on-succes (fn []
                             (rf/dispatch [:chat/save-and-open-emoji-reaction-details])
                             (rf/dispatch [:dismiss-keyboard])
                             (rf/dispatch [:show-bottom-sheet])

```

better to do like this, because its not clear why we pass `drawer-content` into `emoji-reactions-by-message-id` which should just update emoji in app-db

Reply:
> ```clojure
> (rf/dispatch [:chat.ui/emoji-reactions-by-message-id
>                 {:message-id           message-id
>                  :long-pressed-emoji   emoji-id
>                  :on-succes (fn []
>                              (rf/dispatch [:chat/save-emoji-reaction-details])
>                              (rf/dispatch [:dismiss-keyboard])
>                              (rf/dispatch [:show-bottom-sheet])
> ```
> 
> better to do like this, because its not clear why we pass `drawer-content` into `emoji-reactions-by-message-id` which should just update emoji in app-db

```suggestion
(rf/dispatch [:chat.ui/emoji-reactions-by-message-id
                {:message-id           message-id
                 :long-pressed-emoji   emoji-id
                 :on-success (fn []
                             (rf/dispatch [:chat/save-emoji-reaction-details])
                             (rf/dispatch [:dismiss-keyboard])
                             (rf/dispatch [:show-bottom-sheet])
```